GPU/TPCFastTransformation/CorrectionMapsHelper.h

```diff
@@ -119,22 +130,27 @@ class CorrectionMapsHelper
   void setInstLumiOverride(float f) { mInstLumiOverride = f; }
   float getInstLumiOverride() const { return mInstLumiOverride; }
 
+  void setLumiScaleModeOverride(int i) { mLumiScaleModeOverride = i; }
+  int getLumiScaleModeOverride() const { return mLumiScaleModeOverride; }
+
  protected:
   enum UpdateFlags { MapBit = 0x1,
                      MapRefBit = 0x2,
                      LumiBit = 0x4 };
-  bool mOwner = false; // is content of pointers owned by the helper
+  bool mOwner = false;      // is content of pointers owned by the helper
   bool mUseCTPLumi = false; // require CTP Lumi for mInstLumi
   int mUpdatedFlags = 0;
-  float mInstLumi = 0.;                            // instanteneous luminosity (a.u)
-  float mMeanLumi = 0.;                            // mean luminosity of the map (a.u)
-  float mLumiScale = 0.;                           // precalculated mInstLumi/mMeanLumi
-  float mMeanLumiOverride = -1.f;                  // optional value to override mean lumi
-  float mInstLumiOverride = -1.f;                  // optional value to override inst lumi
+  float mInstLumi = 0.;                                         // instanteneous luminosity (a.u)
+  float mMeanLumi = 0.;                                         // mean luminosity of the map (a.u)
+  float mLumiScale = 0.;                                        // precalculated lumi scaling
+  int mLumiScaleMode = 0;                                       // scaling-mode of the correciton maps
+  float mMeanLumiOverride = -1.f;                               // optional value to override mean lumi
+  float mInstLumiOverride = -1.f;                               // optional value to override inst lumi
+  int mLumiScaleModeOverride = 0;                               // optional value to override inst lumi
```

Comment:
The comment is misleading. And this is still not used I think. Do we need it, or can it be removed?

Reply:
The override options for inst and mean lumi are meant to allow overriding values from CTP lumi and mean lumi stored in the map respectively. For the scaling mode, which has no "natural" source, the override is indeed not needed